extensions/spring-xd-extension-jdbc/src/main/java/org/springframework/xd/jdbc/NamedColumnJdbcItemReaderFactory.java

```diff
@@ -59,6 +59,8 @@
 
 	private NamedColumnJdbcItemReader reader;
 
+	private String delimiter;
```

Comment:
I thought the correct way to handle defaults was in the option classes.  If that isn't correct, I can fix...